eng/testing/tests.wasi.targets

```diff
@@ -43,7 +49,7 @@
     <_XHarnessArgs Condition="'$(WasmXHarnessArgs)' != ''"         >$(_XHarnessArgs) $(WasmXHarnessArgs)</_XHarnessArgs>
     <_XHarnessArgs Condition="'$(WasmXHarnessTestsTimeout)' != ''" >$(_XHarnessArgs) &quot;--timeout=$(WasmXHarnessTestsTimeout)&quot;</_XHarnessArgs>
     <_XHarnessArgs Condition="'$(WasmXHarnessArgsCli)' != ''"      >$(_XHarnessArgs) $(WasmXHarnessArgsCli)</_XHarnessArgs>
-    
+
```

Comment:
Adding
```<_XHarnessArgs>$(_XHarnessArgs) --engine-arg=--max-wasm-stack=134217728</_XHarnessArgs>```
fixes stack overflows in the System.Runtime.Serialization.Xml tests.

Reply:
I can add this in a follow up PR.

Reply:
Should this be used for all the aot tests, or only for `System.Runtime.Serialization.Xml`?

Reply:
Also, will this help with oom in System.Globalization.Tests too? https://helixre107v0xdcypoyl9e7f.blob.core.windows.net/dotnet-runtime-refs-pull-95146-merge-7b4bd653d93042109e/System.Globalization.Tests/1/console.cb452bd4.log?helixlogtype=result

Reply:
It should be used for all tests probably. The OOM happens during the aot build, so this will not fix it.